src/pages/iv-ava-by-site/index.js

```diff
@@ -0,0 +1,217 @@
+    const { _startAt = 0, _pageSize = 50, params } = options
+
+    const response = await getRequest({
+      extension: ReportIvGenerator.Report403.endpoint,
+      parameters: `_startAt=${_startAt}&_pageSize=${_pageSize}&_params=${params || ''}&exId=`
+    })
+
+    return { ...response, _startAt: _startAt }
+  }
+
+  async function fetchWithFilter({ filters, pagination }) {
+    return fetchGridData({ _startAt: pagination._startAt || 0, params: filters?.params })
+  }
+
+  const {
+    query: { data },
+    labels: _labels,
+    paginationParameters,
+    refetch,
+    access,
+
+    filterBy
+  } = useResourceQuery({
+    queryFn: fetchGridData,
+    endpointId: ReportIvGenerator.Report403.endpoint,
+    datasetId: ResourceIds.AvailabilitiesBySite,
+    filter: {
+      filterFn: fetchWithFilter
+    }
+  })
+
+  const columns = [
+    {
+      field: 'sku',
+      headerName: _labels.sku,
+      flex: 1
+    },
+    {
+      field: 'name',
+      headerName: _labels.name,
+      flex: 1
+    },
+    {
+      field: 'siteRef',
+      headerName: _labels.siteRef,
+      flex: 1
+    },
+    {
+      field: 'siteName',
+      headerName: _labels.siteName,
+      flex: 1
+    },
+    {
+      field: 'unitCost',
+      headerName: _labels.unitCost,
+      flex: 1
+    },
+    {
+      field: 'unitPrice',
+      headerName: _labels.unitPrice,
+      flex: 1
+    },
+    {
+      field: 'qty',
+      headerName: _labels.qty,
+      flex: 1
+    },
+    {
+      field: 'pieces',
+      headerName: _labels.pieces,
+      flex: 1
+    },
+    {
+      field: 'committed',
+      headerName: _labels.committed,
+      flex: 1
+    },
+    {
+      field: 'netWeight',
+      headerName: _labels.netWeight,
+      flex: 1
+    },
+    {
+      field: 'netVolume',
+      headerName: _labels.netVolume,
+      flex: 1
+    },
+    {
+      field: 'netCost',
+      headerName: _labels.netCost,
+      flex: 1
+    },
+    {
+      field: 'netPrice',
+      headerName: _labels.netPrice,
+      flex: 1
+    },
+    {
+      flex: 1,
+      headerName: 'S/L',
+      cellRenderer: row => {
+        const trackBy1 = row.data.trackBy === 1
+        const trackBy2 = row.data.trackBy === 2
+
+        if (trackBy1) {
+          return (
+            <Box sx={{ display: 'flex', width: '100%', justifyContent: 'center' }}>
+              <IconButton size='small' onClick={() => onSerial(row.data)}>
+                <Image src={serialIcon} width={18} height={18} alt='Serial' />
+              </IconButton>
+            </Box>
+          )
+        }
+
+        if (trackBy2) {
+          return (
+            <Box sx={{ display: 'flex', width: '100%', justifyContent: 'center' }}>
+              <IconButton size='small' onClick={() => onLot(row.data)}>
+                <Image src={lotIcon} width={18} height={18} alt='Lot' />
+              </IconButton>
+            </Box>
+          )
+        }
+
+        return null
+      }
+    }
```

Comment:
maybe if you try this code it will be simpler:

 {
      flex: 1,
      headerName: 'S/L',
      cellRenderer: row => {
        const trackBy1 = row.data.trackBy === 1

        return (
          <Box sx={{ display: 'flex', width: '100%', justifyContent: 'center' }}>
            <IconButton size='small' onClick={() => (trackBy1 ? onSerial(row.data) : onLot(row.data))}>
              <Image src={trackBy1 ? serialIcon : lotIcon} width={18} height={18} alt={serialIcon ? 'Serial' : 'Lot'} />
            </IconButton>
          </Box>
        )
      }
    }

Reply:
and the alt of the buttons should be labels

Reply:
yes its better